src/mds/CInode.h

```diff
@@ -601,7 +601,11 @@ class CInode : public MDSCacheObject, public InodeStoreBase, public Counter<CIno
 
   inodeno_t ino() const { return inode.ino; }
   vinodeno_t vino() const { return vinodeno_t(inode.ino, last); }
+  #ifndef _WIN32
   int d_type() const { return IFTODT(inode.mode); }
+  #else
+  int d_type() const { return -ENOTSUP; }
```

Comment:
as in the other PR, NAK on this

Reply:
> I was a bit concerned that the Windows stat function doesn't set S_IFLNK for symlinks but probably that can be handled somewhere else.

FWIW, it's the same commit. Here's how those PRs were split: http://paste.openstack.org/raw/792975/

Having a single chain of commits (well, apart from those 3 mentioned in the commit description) helped us avoid conflicts between commits affecting the same files, also simplifying the build process.